oneanddone/tasks/serializers.py

```diff
@@ -3,12 +3,18 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 from rest_framework import serializers
 
-from oneanddone.tasks.models import Task, TaskKeyword, TaskAttempt
+from django.contrib.auth.models import User
+
+from oneanddone.tasks.models import (Task, TaskAttempt, TaskKeyword,
+                                     TaskProject, TaskTeam, TaskType)
 
 
 class TaskAttemptSerializer(serializers.ModelSerializer):
 
-    user = serializers.SlugRelatedField(many=False, slug_field='email')
+    user = serializers.SlugRelatedField(
+        many=False,
+        queryset=User.objects.all(),
+        slug_field='email')
```

Comment:
`unique=True` in your model will make sure that you have unique emails. If there are two identical emails in your db, probably the serializer above will produce an error.